README.md

```diff
@@ -8,7 +8,7 @@ This is an Apollo Server integration for Adonisjs 5.
 ## Quickstart
 
 ```sh
-npm install graphql apollo graphql-tag apollo-adonis
+npm install graphql@15.8.0 apollo graphql-tag apollo-adonis
```

Comment:
Do we need to pin this dependecy to this very specific version? Or is it enough to use graphql@15? Or maybe graphql@15.8? Pinning it to 15.8.0 will disable any updates (also security related bug fixes) for this package in the future.

Reply:
Yes my bad, graphql@15.8 is better, there's no need to have this specific version. As long as it's not graphql@16 it should be good !